scripts/build-plugins/rollup-plugin-build-themes.js

```diff
@@ -254,10 +257,6 @@ module.exports = function buildThemes(options) {
                 const derivedVariables = compiledVariables["derived-variables"];
                 const icon = compiledVariables["icon"];
                 const builtAssets = {};
-                /**
-                 * Generate a mapping from theme name to asset hashed location of said theme in build output.
-                 * This can be used to enumerate themes during runtime.
-                 */
                 for (const chunk of chunkArray) {
                     const [, name, variant] = chunk.fileName.match(/theme-(.+)-(.+)\.css/);
                     builtAssets[`${name}-${variant}`] = assetMap.get(chunk.fileName).fileName;
```

Comment:
Not modified in this PR, but I was thinking if this path shouldn't be relative to the theme manifest ...
Right now it is relative to the build root, (which can be deployed on the document root or anywhere deeper on a server).

When you have the config file refer to theme manifests on a different server, it's unclear how the absolute URL of the css files should be resolved if these paths are not relative to the theme manifest, as you can't really assume that there is another hydrogen deployment in the parent directory with an assets directory on the other server.